docs/source/getting-started/registration.rst

```diff
@@ -23,4 +24,4 @@ And then save the generated certificate to a file
 .. code:: python
 
     with open('cert.json', 'w') as f: # You can use other filename
-        json.dump(certificate.json, f)
```

Comment:
This was good, it needs to use `json` property, because we want to save json. Please, look at that line https://github.com/kapi2289/vulcan-api/blob/master/vulcan/_certificate.py#L20

Reply:
![2](https://user-images.githubusercontent.com/59137941/74085677-e284ac80-4a7b-11ea-94f6-69538bcd2be4.PNG)
That is the output of the example code from the current documentation. I think it's because 'Vulcan.register' returns 'Certificate.get()' which returns a dictionary...
`        cert = j["TokenCert"]
        log.info("Registered successfully!")

        return cert
`

Reply:
Oh, that's my fault! It should return a Certificate object, I'll fix that.

Reply:
Or maybe I'll just leave it as it is...